README.md

```diff
@@ -195,6 +195,10 @@ The following table provides a short summary of available features for supported
     <td><a href="./docs/usage.md#generate-models-from-asyncapi-documents">AsyncAPI</a></td>
     <td>We support the following AsyncAPI versions: <em>2.0.0 -> 2.6.0</em>, which generates models for all the defined message payloads. It supports the following schemaFormats AsyncAPI Schema object, JSON Schema draft 7, <a href="./examples/asyncapi-avro-schema">AVRO 1.9</a>, <a href="./examples/asyncapi-raml-schema">RAML 1.0 data type</a>, and <a href="./examples/asyncapi-openapi-schema">OpenAPI 3.0 Schema</a>.</td>
   </tr>
+  <tr>
+    <td><a href="./docs/usage.md#generate-models-from-avro-schema-documents">Avro Schema<a></td>
+    <td>we support the Avro Schema version <em>1.11.1</em> to generate models</td>
```

Comment:
No need to be that explicit for the version, as we technically support all versions, cause we dont check it anywhere 🤔 
```suggestion
    <td>We support the following Avro versions: <a href="./examples/avro-schema-from-object">v1.x</a></td>
```